generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -21,6 +21,12 @@ class TinyRocketConfig extends Config(
   new freechips.rocketchip.subsystem.With1TinyCore ++             // single tiny rocket-core
   new chipyard.config.AbstractConfig)
 
+class RocketWithFFT extends Config(
```

Comment:
This should be `FFTRocketConfig`, and it should branch off of `AbstractConfig`.

Reply:
+1